File: app/src/test/java/com/timewasteanalyzer/util/UtilityTest.java
package com.timewasteanalyzer.util;


import org.junit.Test;

import static junit.framework.Assert.assertFalse;
import static junit.framework.Assert.assertTrue;


public class UtilityTest {

    @Test
    public void emptyString() {
        assertTrue(Utility.INSTANCE.isEmpty(""));
    }

    @Test
    public void nullString() {
        assertTrue(Utility.INSTANCE.isEmpty(null));
    }

    @Test
    public void validString() {
        assertFalse(Utility.INSTANCE.isEmpty("some text"));
    }

}

File: app/src/test/kotlin/com/timewasteanalyzer/util/UtilityTest.kt
package com.timewasteanalyzer.util


import junit.framework.Assert.assertFalse
import junit.framework.Assert.assertTrue
import org.junit.Test


class UtilityTest {

    @Test
    fun equalsEmptyStrings() {
        assertTrue(Utility.equals("", ""))
    }

    @Test
    fun equalsStrings() {
        assertTrue(Utility.equals("a", "a"))
    }

    @Test
    fun equalsFirstStringEmpty() {
        assertFalse(Utility.equals("", "a"))
    }

    @Test
    fun equalsFirstStringNull() {
        assertFalse(Utility.equals(null, "a"))
    }

    @Test
    fun equalsSecondStringEmpty() {
        assertFalse(Utility.equals("a", ""))
    }

    @Test
    fun equalsSecondStringNull() {
        assertFalse(Utility.equals("a", null))
    }

    @Test
    fun emptyString() {
        assertTrue(Utility.isEmpty(""))
    }

    @Test
    fun nullString() {
        assertTrue(Utility.isEmpty(null))
    }

    @Test
    fun validString() {
        assertFalse(Utility.isEmpty("some text"))
    }

}
